tests/python/unittest/test_gluon_estimator.py

```diff
@@ -332,39 +332,41 @@ def test_default_handlers():
     train_acc = mx.metric.RMSE()
     loss = gluon.loss.L2Loss()
 
+    gradient_update = GradientUpdateHandler()
     est = Estimator(net=net,
                     loss=loss,
                     metrics=train_acc,
                     trainer=trainer,
                     context=ctx)
-    # no handler(all default handlers), no warning
+    # no handler except gradient update handler (all default handlers), no warning
     with warnings.catch_warnings(record=True) as w:
-        est.fit(train_data=train_data, epochs=num_epochs)
+        est.fit(train_data=train_data, epochs=num_epochs, event_handlers=[gradient_update])
```

Comment:
you don't need to add `gradient_update` here right? as it's added as a default handler?

Reply:
My mistake. I'll remove this handler here.